### PR TITLE
add truncate module docs

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: search/truncate/truncate.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## truncate 
+
+The `truncate` module preserves only the first N characters (or bytes when using binary mode) of enumerated values. For example, to truncate all but the first 20 characters of the EV "Message":
+
+```
+tag=data json IP Message | truncate -e Message 20 | table
+```
+
+`truncate` only operates string and byte slice enumerated values and by default assumes data is UTF-8 encoded. You can override this behavior by using the `-binary` flag.
```
Comment (floren) [inline search/truncate/truncate.md:9]: should be "operates on string"

File: search/truncate/truncate.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Supported Options
+
+* `-ellipsis`: Optional. Add an ellipsis (three character ...) to the string after truncating, meaning all truncated values will be three characters longer than specified.
```
Comment (floren) [inline search/truncate/truncate.md:13]: change parenthetical to read `(three period characters, "...")`

Comment (floren) [reply search/truncate/truncate.md:13]: Also emphasize "after", `*after*`

**Comment by djfritz** [comment]
@floren PTAL

**Review by floren — APPROVED** [review]

**Pull request merged**